README.md

```diff
@@ -88,8 +87,8 @@ Chapter 3 - Vectors
 Chapter 4 - Subsetting
 
 - Cohort 1: [Scott Nestler](https://twitter.com/ScottNestler) ([slides](https://r4ds.github.io/bookclub-Advanced_R/Presentations/Week04/Cohort1/Chap4slides.html#1) | [video](https://www.youtube.com/watch?v=eLMpCc0t1cg))
-
-Chapter 5 - Control flow
```

Comment:
Hmm. Looks like you dropped this chapter five line

Reply:
I think I fixed that in the second commit.